docs/source/tutorial/tutorial-introduction.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "0. Introduction"
+---
+
+Welcome! This tutorial demonstrates adding the Apollo iOS SDK to an app to communicate with a GraphQL server. It was prepared with the following tools:
+
+- Xcode 11.2
+- Swift 5.1
+- Apollo iOS SDK 0.20.0
+
+The tutorial assumes that you're using a Mac with Xcode installed. It also assumes some prior experience with iOS development.
+
+> If you encounter any issues during the tutorial, feel free to ask questions by either [opening an issue on our GitHub repo](https://github.com/apollographql/apollo-ios/issues) or [stopping by our Spectrum Chat for help](https://spectrum.chat/apollo/apollo-ios).
+
+## What are you building?
+
+This tutorial takes you through building an app to "book a seat" on any of the rockets that have been sent into space by [SpaceX](https://www.spacex.com/). Along the way, you'll build out functionality that uses the following features of the Apollo iOS SDK:
+
+* Schema downloading
+* Code generation
+* Queries and mutations
+* Error handling
+* Query watching
+* Subscriptions
+* File uploads
+
+A pre-built GraphQL server is provided for your app to execute operations against.
+
+## Ready to start?
+
+[🚀 Let's go! 🚀](./tutorial-create-project)
```

Comment:
why does this need `./` instead of the name of the file?

Reply:
This is purely a personal preference thing; all links between articles in the same docset should be relative to ensure that the link checker can identify broken ones, and this looks more like a relative path to me

Reply:
OK, even though functionally `(tutorial-create-project)` is the same?